### ISSUE TITLE
Wrong menu item selected

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#148158]
* vscode.insiders, safari
* trigger quick fix for a missing import
* for a split second the first menu entry is selected (the correct one) but than the last is selected (happens at ~sec 2 in the recording)

https://user-images.githubusercontent.com/1794099/165321268-9c067346-48f3-4d3e-8123-b8bcac46e425.mov
<img width="597" alt="Screenshot 2022-04-26 at 16 19 40" src="https://user-images.githubusercontent.com/1794099/165321283-ce7590c5-82d3-4270-800d-5af67b068890.png">

**Comment by sbatten** [comment]
@jrieken is your mouse cursor over that area when using the keyboard shortcut? trying to discern if its caused by mouse events

**Comment by jrieken** [comment]
Yeah, very likely. That's how it reproduces at least, tho the mouse is just sitting there, not being moved/touched.
